radio/src/targets/flysky/crc_driver.cpp

```diff
@@ -13,7 +13,9 @@ void crcInit() {
 }
 
 uint8_t crc8_hw(const uint8_t * ptr, uint32_t len) {
+  CRC->INIT = CRC8_INIT_VAL;
```

Comment:
After this change crcInit is not needed and must be removed.

Reply:
Even if crc16 doesn't use the same init value? CRC uses 0xFFFF.

Don't we need to change it back every time the code runs either crc8 or crc16?

Reply:
It is now set up in every crc function so initCrc does not make sense

Reply:
Ohh I get it, sorry I originally misread your comment half way through the night. All fixed now

Reply:
Remove it completely instead of commenting out, not needed.